intelmq/bots/BOTS

```diff
@@ -4,6 +4,8 @@
             "description": "Fileinput collector fetches data from a file.",
             "module": "intelmq.bots.collectors.file.collector_file",
             "parameters": {
+                "chunk_replicate_header": true,
```

Comment:
As written in https://github.com/certtools/intelmq/pull/680#issuecomment-274758167 not every collector might be capable of splitting the data. It depends on the data-format if it can be split.
CSV or some Blocklists can be split this way, whilst XML, JSON, Binary cannot!